cpp/perspective/CMakeLists.txt

```diff
@@ -250,6 +250,7 @@ if (PSP_WASM_BUILD)
 			-s DISABLE_EXCEPTION_CATCHING=0 \
 			-s ASSERTIONS=2 \
 			-s DEMANGLE_SUPPORT=1 \
+			-s SAFE_HEAP=1 \
```

Comment:
IIRC this setting basically makes debug-build Perspective unusable (though it is quite useful for finding memory leaks!)